lib/libzfs/libzfs_sendrecv.c

```diff
@@ -277,7 +277,7 @@ cksummer(void *arg)
 		{
 			struct drr_begin *drrb = &drr->drr_u.drr_begin;
 			int fflags;
-			int sz = 0;
+			size_t sz = 0;
```

Comment:
@behlendorf 
Communicate with you through this code.
I am very happy.
Your exact and friendly, Make me feel very kind.
thank you.